src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -576,7 +567,49 @@ int fetch_object_timeout_handler(event_loop *loop, timer_id id, void *context) {
              end_time - start_time);
   }
 
-  return LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS;
+  /* Wait at least LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS before running
```

Comment:
rename LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS to kLocalSchedulerFetchTimeoutMilliseconds